tools/common/StringUtils.cs

```diff
@@ -55,5 +55,21 @@ public static string Unquote (string input)
 			return builder.ToString ();
 		}
 
+		// Version.Parse requires, minimally, both major and minor parts.
+		// However we want to accept `11` as `11.0`
+		public static Version ParseVersion (string v)
+		{
+			try {
+				// assume the normal case
+				return Version.Parse (v);
```

Comment:
I don't like to have exceptions in the normal/accepted workflow (at the very least it's annoying when debugging and breaking on all exceptions; it's also quite slow), maybe something like this instead?

```csharp
int major;
if (int.TryParse (v, out major))
    return new Version (major, 0);
return Version.Parse (v);
```

Reply:
There should not be exceptions in the normal/default code path (IDE provide the extra `.0`) but there's no harm in having faster, exception-less, code everywhere :)